<ui version="4.0" >
 <class>PDFDocument</class>
 <widget class="QMainWindow" name="PDFDocument" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout_QTeX" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionOpen_Recent" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
    <addaction name="separator" />
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
    <addaction name="actionClear" />
    <addaction name="separator" />
    <addaction name="actionPreferences" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionFirst_Page" />
    <addaction name="actionPrevious_Page" />
    <addaction name="actionNext_Page" />
    <addaction name="actionLast_Page" />
    <addaction name="separator" />
    <addaction name="actionGo_to_Page" />
    <addaction name="separator" />
    <addaction name="actionActual_Size" />
    <addaction name="actionFit_to_Width" />
    <addaction name="actionFit_to_Window" />
    <addaction name="actionZoom_In" />
    <addaction name="actionZoom_Out" />
   </widget>
   <widget class="QMenu" name="menuWindow" >
    <property name="title" >
     <string>Window</string>
    </property>
    <addaction name="actionStack" />
    <addaction name="actionTile" />
    <addaction name="actionTile_Front_Two" />
    <addaction name="separator" />
    <addaction name="actionGo_to_Source" />
   </widget>
   <widget class="QMenu" name="menuTypeset" >
    <property name="title" >
     <string>Typeset</string>
    </property>
    <addaction name="actionTypeset" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuView" />
   <addaction name="menuTypeset" />
   <addaction name="menuWindow" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="iconSize" >
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionFirst_Page" />
   <addaction name="actionPrevious_Page" />
   <addaction name="actionNext_Page" />
   <addaction name="actionLast_Page" />
   <addaction name="separator" />
   <addaction name="actionZoom_In" />
   <addaction name="actionZoom_Out" />
   <addaction name="actionActual_Size" />
  </widget>
  <action name="actionAbout_QTeX" >
   <property name="text" >
    <string>About TeXWorks...</string>
   </property>
  </action>
  <action name="actionFirst_Page" >
   <property name="text" >
    <string>First Page</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionPrevious_Page" >
   <property name="text" >
    <string>Previous Page</string>
   </property>
   <property name="shortcut" >
    <string>Left</string>
   </property>
  </action>
  <action name="actionNext_Page" >
   <property name="text" >
    <string>Next Page</string>
   </property>
   <property name="shortcut" >
    <string>Right</string>
   </property>
  </action>
  <action name="actionLast_Page" >
   <property name="text" >
    <string>Last Page</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="action" >
   <property name="text" >
    <string>-</string>
   </property>
  </action>
  <action name="actionGo_to_Page" >
   <property name="text" >
    <string>Go to Page...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionZoom_In" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/zoomin.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom In</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/zoomout.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Out</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionFit_to_Window" >
   <property name="text" >
    <string>Fit to Window</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionActual_Size" >
   <property name="text" >
    <string>Actual Size</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionFit_to_Width" >
   <property name="text" >
    <string>Fit to Width</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="text" >
    <string>New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Recent" >
   <property name="text" >
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="text" >
    <string>Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="text" >
    <string>Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="text" >
    <string>Cut</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="text" >
    <string>Copy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="text" >
    <string>Paste</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClear" >
   <property name="text" >
    <string>Clear</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionTypeset" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/images/images/typeset.png</iconset>
   </property>
   <property name="text" >
    <string>Typeset</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionPreferences" >
   <property name="text" >
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionStack" >
   <property name="text" >
    <string>Stack</string>
   </property>
  </action>
  <action name="actionTile" >
   <property name="text" >
    <string>Tile</string>
   </property>
  </action>
  <action name="actionTile_Front_Two" >
   <property name="text" >
    <string>Tile Front Two</string>
   </property>
  </action>
  <action name="actionGo_to_Source" >
   <property name="text" >
    <string>Go to Source</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+'</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>PDFDocument</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
